### ISSUE TITLE
Make helm template work with remote repositories

### ISSUE THREAD

**Issue by Kronuz** [helm/helm#4527]
`helm template stable/rabbitmq` should fetch and produce the expected rendered template (from the charts repository); instead, I get: `Error: stat stable/rabbitmq: no such file or directory`

Output of `helm version`:
```
Client: &version.Version{SemVer:"v2.10.0", GitCommit:"9ad53aac42165a5fadc6c87be0dea6b115f93090", GitTreeState:"clean"}
```
Output of `kubectl version`:
```
Client Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.7", GitCommit:"dd5e1a2978fd0b97d9b78e1564398aeea7e7fe92", GitTreeState:"clean", BuildDate:"2018-04-19T00:05:56Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"10+", GitVersion:"v1.10.6-gke.2", GitCommit:"384b4eaa132ca9a295fcb3e5dfc74062b257e7df", GitTreeState:"clean", BuildDate:"2018-08-15T00:10:14Z", GoVersion:"go1.9.3b4", Compiler:"gc", Platform:"linux/amd64"}
```
Cloud Provider/Platform (AKS, GKE, Minikube etc.): `GKE`

**Comment by RehanSaeed** [comment]
Was wondering why this was not working until I saw this issue. The workaround I found for this issue is to use `helm fetch` to download the chart and then run it from a file:

```
helm fetch --untar --untardir . 'stable/redis'
helm template --output-dir './redis-final' './redis'
```

**Comment by bacongobbler** [comment]
this is being fixed in the Helm 3 branch via https://github.com/helm/helm/pull/5171 if I'm not mistaken. The core maintainers are busy working on Helm 3 development so if someone wants to make a patch to Helm 2, we'd appreciate it.

**Comment by procyclinsur** [comment]
made a shell function of the workaround provided by @RehanSaeed for anyone interested.

requires: vim

```bash
function helm-tmpl-oride() {
    ###########
    # Variables
    ###########
    RLSE=$1  # Helm release name
    CHRT=$2  # Helm chart name
    FILE=$3  # File path/name
    BASE=$PWD
    
    #############
    # Fetch Chart
    #############
    mkdir -p /tmp/helm
    cd /tmp/helm
    helm fetch --untar --untardir . $CHRT
    CHRT_DIR=$(ls -td /tmp/helm/*/ | head -n 1)
    
    ################
    # Check template
    ################
    helm template --name $RLSE $CHRT_DIR -f $BASE/$FILE > manifest.yml
    vim manifest.yml
    cd $BASE
}
```

**Comment by bacongobbler** [comment]
Fixed in Helm 3 via #5171. Closing!
